chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -100,13 +120,14 @@ Zotero.QuickCopy = new function() {
 			+ "WHERE setting='quickCopySite' AND (key LIKE ? OR key LIKE ?)";
 		var urlDomain = urlHostPort.match(/[^\.]+\.[^\.]+$/);
 		var rows = Zotero.DB.query(sql, ['%' + urlDomain + '%', '/%']);
-		for each(var row in rows) {
-			var [domain, path] = row.domainPath.split(/\//);
+		for (let i = 0; i < rows.length; i++) {
+			let row = rows[i];
+			let [domain, path] = row.domainPath.split(/\//);
 			path = '/' + (path ? path : '');
-			var re = new RegExp(domain + '$');
-			if (urlHostPort.match(re) && urlPath.indexOf(path) == 0) {
+			let re = new RegExp(domain + '$');
```

Comment:
BTW, I'm just mostly making notes here for myself, so I remember this tomorrow. Sorry, @rmzelle, for the noise